playground/mocks/data/idp/idx/error-authenticator-enroll-security-question-create-question.json

```diff
@@ -1,65 +1,133 @@
+                            "value":"childhood_dream_job"
+                          },
+                          {
+                            "label":"Where did you meet your spouse/significant other?",
+                            "value":"place_where_significant_other_was_met"
+                          },
+                          {
+                            "label":"Where did you go for your favorite vacation?",
+                            "value":"favorite_vacation_location"
+                          },
                           {
-                            "label": "What is the food you least liked as a child?",
-                            "value": "disliked_food"
+                            "label":"Where were you on New Year's Eve in the year 2000?",
+                            "value":"new_years_two_thousand"
                           },
                           {
-                            "label": "What is the name of your first stuffed animal?",
-                            "value": "name_of_first_plush_toy"
+                            "label":"Who is your favorite speaker/orator?",
+                            "value":"favorite_speaker_actor"
                           },
                           {
-                            "label": "Where did you go for your favorite vacation?",
-                            "value": "favorite_vacation_location"
+                            "label":"Who is your favorite book/movie character?",
+                            "value":"favorite_book_movie_character"
+                          },
+                          {
+                            "label":"Who is your favorite sports player?",
+                            "value":"favorite_sports_player"
                           }
                         ]
                       },
                       {
-                        "name": "answer",
-                        "label": "Answer",
-                        "required": true,
-                        "secret": true,
-                        "messages": {
-                          "type": "array",
-                          "value": [
+                        "name":"answer",
+                        "label":"Answer",
+                        "required":true,
+                        "messages":{
+                          "type":"array",
+                          "value":[
                             {
-                              "class": "ERROR",
-                              "i18n": {
-                                "key": "securityQuestion.answer.tooShort",
-                                "params": []
+                              "message":"The security question answer must be at least 4 characters in length",
+                              "i18n":{
+                                "key":"securityQuestion.answer.tooShort.arg",
+                                "params":[
+                                  4
+                                ]
```

Comment:
It seems that there is no more `securityQuestion.answer.tooShort` and instead `securityQuestion.answer.tooShort.arg` with params coming from backend. I have discussed in detail with @rameshbhojan-okta .

`securityQuestion.answer.tooShort` seems to be a V1 thing now.

Hence updated this mock and the next one as well.

Reply:
I don't remember any recent changes w.r.t this and AFAIK this was always the case - not sure why mocks here in SIW were different all this while. 

`securityQuestion.answer.tooShort` was used only in the property level validation from MFA Question in V1. 

> It seems that there is no more securityQuestion.answer.tooShort and instead securityQuestion.answer.tooShort.arg with params coming from backend. I have discussed in detail with @rameshbhojan-okta .
> 
> securityQuestion.answer.tooShort seems to be a V1 thing now.